main/util-linux/APKBUILD

```diff
@@ -82,68 +77,59 @@ dev() {
 }
 
 bashcomp() {
-	depends=""
```

Comment:
This should not be removed. It is necessary to reset `depends` manually in order to not inherit the parent package's manual dependencies. Same applies for other split functions and `replaces`.

Reply:
I fixed it.